jib-cli/src/test/java/com/google/cloud/tools/jib/cli/war/WarFilesTest.java

```diff
@@ -144,29 +143,9 @@ public void testToJibContainerBuilder_nonJettyBaseImageSpecifiedAndNoEntrypoint(
             mockLogger);
     ContainerBuildPlan buildPlan = containerBuilder.toContainerBuildPlan();
 
-    assertThat(buildPlan.getBaseImage()).isEqualTo("base-image");
     assertThat(buildPlan.getEntrypoint()).isNull();
```

Comment:
Hmm... does it make sense that the entrypoint is null when `isJetty()` from the mock is returning true?

Reply:
I added an assert statement to clarify this. Let me know what you think?

Reply:
Oh, my bad. I mistakenly thought the new addition on line 65 was for `setUp()` to affect every test.